### DISCUSSION TITLE
Resources Loaded

### DISCUSSION THREAD

**Discussion by iBotPeaches** [umputun/remark42#975] [General]
Hi there,

Just once again chasing that 100% pagespeed and have a few questions.

1. I notice this web/remark42.mjs files makes two requests. As far as I can tell those files are different, but named the same. Do you have any background for my curiosity on the difference on these?
![files](https://user-images.githubusercontent.com/611784/117136169-02397c00-ad76-11eb-8e1d-92e078b6b4f1.png)

2. The mjs file(s) seem to be minified, but the remark.css file is not. Do you see any gain/benefit of also minifying that one? I see an old commit that looks to be minifying resources, but maybe I misread it - https://github.com/umputun/remark42/commit/bb5907d21f3200a51772a2cc2da8f2b7a85b7952

thanks thanks!

**Comment by akellbl4** [comment]
Hi, we really care about the bundle size of JS and CSS.
Not so long ago we started to use es modules for modern browsers. It made our bundle smaller

1. Minifying CSS file is very important in my opinion and I don't know when we lost it for our CSS (maybe we don't have it in the first place).  We will definitely minify CSS bundle.
  UPD. [here is a fix](https://github.com/umputun/remark42/pull/977)

2.  Along with modern build, we use `modulereload` you can read about [here](https://developers.google.com/web/updates/2017/12/modulepreload) And as I can see I totally forgot to add `type="module"` on the script. It causes the second request. Thank you for reporting that.
  UPD. [here is a fix](https://github.com/umputun/remark42/pull/976)

**Reply by iBotPeaches** [reply]
Thank you for the fast turn around and even fix! Will look forward to v1.9

**Reply by akellbl4** [reply]
Additionally, I've shortened class names it made the bundle even smaller
https://github.com/umputun/remark42/pull/980#issuecomment-832946888
